tests: crypto:  mcxw72_evk /  frdm_mcxw72: build failure

**Describe the bug**

mcxw72_evk and frdm_mcxw72  below test build failure with
tests/subsys/jwt/
tests/subsys/secure_storage
tests/subsys/random/rng

```
fatal error: sss_crypto.h: No such file or directory
```

**To Reproduce**


```
west twister -p mcxw72_evk/mcxw727c/cpu0 -T tests/drivers/entropy/api  -s drivers.entropy
```
or with west:
```
west build -p -b mcxw72_evk/mcxw727c/cpu0 tests/drivers/entropy/api
```

**Expected behavior**

Build pass

**Impact**

CI for mcxw72_evk/mcxw727c/cpu0 , frdm_mcxw72/mcxw727c/cpu0

**Logs and console output**

```
zephyr/drivers/entropy/CMakeFiles/drivers__entropy.dir/entropy_nxp_ele.c.obj -MF zephyr/drivers/entropy/CMakeFiles/drivers__entropy.dir/entropy_nxp_ele.c.obj.d -o zephyr/drivers/entropy/CMakeFiles/drivers__entropy.dir/entropy_nxp_ele.c.obj -c /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/drivers/entropy/entropy_nxp_ele.c\n/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/drivers/entropy/entropy_nxp_ele.c:18:10: fatal error: sss_crypto.h: No such file or directory\n   18 | #include "sss_crypto.h"\n      |          ^~~~~~~~~~~~~~\ncompilation terminated.\nninja: build stopped: subcommand failed.\n', 'TEST_PATH': 'testing.ztest.base.verbose_1'}]]}
```

**Environment (please complete the following information):**

 - OS: (e.g. Linux, MacOS, Windows)
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used: v4.0.0-5520-gc544e4d08b43

--- a/west.yml
+++ b/west.yml
@@ -203,7 +203,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 605560a680318ec417c1ee38d05ac092c383dc1d
+      revision: 2a1d73aeb863a73bb06ba8b236de7da6d3e31847
       path: modules/hal/nxp
       groups:
         - hal

